tools/DevDiagnostics/DevHome.DevDiagnostics/App.config

```diff
@@ -1,71 +1,71 @@
-            <section name="DevHome.DevInsights.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false" />
-        </sectionGroup>
-    </configSections>
-    <userSettings>
-      <DevHome.DevInsights.Properties.Settings>
-        <setting name="IsProcessFilterIncludeConHost" serializeAs="String">
-          <value>False</value>
-        </setting>
-        <setting name="IsProcessFilterIncludeDllHost" serializeAs="String">
-          <value>False</value>
-        </setting>
-        <setting name="IsProcessFilterIncludeSvcHost" serializeAs="String">
-          <value>False</value>
-        </setting>
-        <setting name="IsProcessFilterIncludeEdge" serializeAs="String">
-          <value>False</value>
-        </setting>
-        <setting name="IsProcessFilterIncludeWebview" serializeAs="String">
-          <value>False</value>
-        </setting>
-        <setting name="IsProcessFilterIncludeRtb" serializeAs="String">
-          <value>False</value>
-        </setting>
-        <setting name="IsProcessFilterIncludeWmi" serializeAs="String">
-          <value>False</value>
-        </setting>
-        <setting name="IsProcessFilterIncludeWudf" serializeAs="String">
-          <value>False</value>
-        </setting>
-        <setting name="IsProcessFilterIncludeBgTaskHost" serializeAs="String">
-          <value>False</value>
-        </setting>
-        <setting name="CurrentTheme" serializeAs="String">
-          <value>Default</value>
-        </setting>
-        <setting name="IsCpuUsageMonitoringEnabled" serializeAs="String">
-          <value>True</value>
-        </setting>
-        <setting name="IsInsightsOnStartupEnabled" serializeAs="String">
-          <value>False</value>
-        </setting>
-        <setting name="IsClipboardMonitoringEnabled" serializeAs="String">
-          <value>True</value>
-        </setting>
-        <setting name="ExcludedProcesses" serializeAs="Xml">
-          <value>
-            <ArrayOfString xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-              <string>DevHome.DevInsights</string>
-              <string>DevEnv</string>
-            </ArrayOfString>
-          </value>
-        </setting>
-        <setting name="ExpandedWindowHeight" serializeAs="String">
-          <value>0</value>
-        </setting>
-        <setting name="ApplyAppFilteringToData" serializeAs="String">
-          <value>True</value>
-        </setting>
-        <setting name="WindowWidth" serializeAs="String">
-          <value>0</value>
-        </setting>
-        <setting name="IsClipboardMonitorToolPinned" serializeAs="String">
-          <value>True</value>
-        </setting>
-      </DevHome.DevInsights.Properties.Settings>
-    </userSettings>
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Looks like a bunch of files got their line endings changed. This one at least is LF now, should be put back to CRLF.

Reply:
Hmm... I'm seeing CRLF in this file and others...???